template/lambda-sqs-worker-cdk/package.json

```diff
@@ -4,6 +4,8 @@
   "scripts": {
     "build": "skuba build",
```

Comment:
Does this render `skuba build` redundant for this template?

Reply:
Ruh roh looks like removing this causes issues with test-template

Reply:
Heh, I think we can catch and ignore "Command not found" errors there 🙏